examples/operation-security.yml

```diff
@@ -0,0 +1,106 @@
+asyncapi: 2.3.0
+info:
+  title: Notifications
+  version: 1.0.0
+  description: >-
+    This contract defines HTTP Push notification for 
+    application authorization revocation topic
+channels:
+  AUTHORIZATION_REVOCATION:
+    subscribe:
+      message:
+        $ref: '#/components/messages/message'
+      bindings:
+        http:
+          type: request
+          method: POST
+          headers:
+            type: object
+            properties:
+              Content-Type:
+                type: string
+                enum:
+                  - application/json
+      security:
+        petstore_auth:
```

Comment:
In this example, `petstore_auth` is a key in the `security` object but the spec (and the examples on the spec) say it should be an array like:

```yaml
security:
  - petstore_auth:
    - subscribe:auth_revocations
```

Honestly, I think this example is correct and we should change the spec description because [`security` is also a key in the Server Object](https://www.asyncapi.com/docs/specifications/v2.3.0#securityRequirementObject). We should be consistent with the Server Object IMHO.

Reply:
Hi @fmvilas  - I see your first point - yes the example has it wrong. But to me, the proposed spec description is correct  and does have security: [[Security Requirement Object](https://github.com/asyncapi/spec/blob/3460840ee5a68b0b9f6d881da8f6262cc8602ef2/spec/asyncapi.md#securityRequirementObject)]

which is the same as on the server level. I ll fix the example.  Apologize if i misunderstood your comment

Reply:
You're right, I got confused by the Security Requirement Object examples 🤦 It should be an array of objects 👍